odpf/optimus/runtime_service.proto

```diff
@@ -168,6 +168,11 @@ service RuntimeService {
             body: "*"
         };
     }
+    rpc ReplayDryRun(ReplayDryRunRequest) returns (ReplayDryRunResponse) {
+        option (google.api.http) = {
+            get: "/api/v1/project/{project_name}/job/{job_name}/replayDryRun"
```

Comment:
Can we use lower case letters with hyphen(-) separator(spinal-case) here?

Reply:
updated